Maestro/subscriptions.json

```diff
@@ -82,6 +82,20 @@
             "-GitHubPullRequestNotifications 'dotnet/corefx-contrib'",
             "-DirPropsVersionElements CoreFxExpectedPrerelease"
           ]
+        },
+        // This handler will bring the Latest CoreFX master build into CoreCLR master
+        {
+          "maestroAction": "coreclr-general",
+          "maestroDelay": "00:10:00",
+          "ScriptFileName": "UpdateDependencies.ps1",
+          "Arguments": [
+            "-GitHubUser dotnet-bot",
+            "-GitHubEmail dotnet-bot@microsoft.com",
+            "-GitHubPassword `$(`$Secrets['DotNetBotGitHubPassword'])",
+            "-VersionFileUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master/Latest.txt",
+            "-GitHubPullRequestNotifications 'dotnet/coreclr-contrib'",
+            "-DirPropsVersionElements CoreFxExpectedPrerelease"
```

Comment:
I don't see either `CoreFxExpectedPrerelease` or `CoreClrExpectedPrerelease` in [https://github.com/dotnet/coreclr/blob/master/dir.props](https://github.com/dotnet/coreclr/blob/master/dir.props).  I only see `<ExternalExpectedPrerelease>beta-24319-00</ExternalExpectedPrerelease>`, which already has a handler in this file.

Are we sure this is the right handler?